lib/airtable/table.rb

```diff
@@ -71,8 +93,8 @@ def result_attributes(res)
     end
 
     def worksheet_url
-      "/#{app_token}/#{URI.encode(worksheet_name)}"
+      "/#{app_token}/#{CGI.escape(worksheet_name)}"
```

Comment:
What's the difference?

Side note: it'd be very nice to rename "worksheet" to "table" in the client.